### ISSUE TITLE
Update all instances of Mozilla logo and wordmark

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#15682]
There are a few places using old brand assets as part of other images. We should find and update them.

Jan started [a good list in the comments](https://github.com/mozilla/bedrock/issues/15416#issuecomment-2456101249) here.

Also, note: open graph / share images should be part of the search.

**Comment by janbrasna** [comment]
- `media/img/newsletter/confirm/mozilla.png` on `newsletter/updated`:

<img width="513" alt="Screenshot 2024-12-05 at 2 38 50" src="https://github.com/user-attachments/assets/2a934927-a428-484e-bf1f-70d0bc2f5a0b">

**Comment by janbrasna** [comment]
These "m" icons will need replacing with dino flag, too, right? (This is just one instance, there will be more…)

- `media/img/trademarks/logo-mozm.png` in `privacy/mozilla-*`:

<img width="491" alt="Screenshot 2024-12-05 at 3 17 48" src="https://github.com/user-attachments/assets/44f5b703-2039-4e1e-a831-33df69338fd8">

**Comment by stephaniehobson** [comment]
@janbrasna yes, the "m" should become the green dino flag (like the favicon)

**Comment by maureenlholland** [comment]
should be completed with https://github.com/mozilla/bedrock/pull/15975

**Comment by janbrasna** [comment]
There's one additional "m" occurrence: https://www.mozilla.org/firefox/set-as-default/thanks/

> ![moz-m](https://github.com/user-attachments/assets/2dd7a85c-7e66-42a9-89fd-eb7a3ae531c4)

https://github.com/mozilla/bedrock/blob/823fae6b4ba3df40b73dbf4d2a30d60ddd3ade87/media/css/firefox/set-as-default/thanks.scss#L110-L113

What should an FxA "Create an account" call-to-action have as an icon? Flag symbol? Or a "sync" metaphor arrows?

**Comment by stephaniehobson** [comment]
Re-opening because of the 2 additional logos Jan found (thanks!)

**Comment by janbrasna** [comment]
FYI the 404 icon is pure Protocol so that one gets updated to flag symbol with v21 (see [fxc 404](https://www-dev.springfield.moz.works/en-US/404/) how that looks…)

There's one more in https://www.mozilla.org/firefox/features/tips/ just above footer which also taps into `protocol-assets` social icons so that one will get updated with MZP bump too (see [fxc](https://www-dev.springfield.moz.works/features/tips/) already updated…)

**Comment by adi-ray** [comment]
> > ![moz-m](https://github.com/user-attachments/assets/2dd7a85c-7e66-42a9-89fd-eb7a3ae531c4)
>
> 
> What should an FxA "Create an account" call-to-action have as an icon? Flag symbol? Or a "sync" metaphor arrows?

bump @stephaniehobson

**Comment by janbrasna** [comment]
Exactly, a bump will take care of that. These will come "for free" when updated to Protocol v21/22…

(If you want to take a stab at the last remaining one, it's actually already moved to [`@mozmeao/springfield`](https://github.com/mozmeao/springfield) — the https://www.firefox.com/landing/set-as-default/thanks/ can now have the CTA trio logo changed to a Protocol asset ([protocol/img/logos/mozilla/logo.svg](https://www.firefox.com/media/protocol/img/logos/mozilla/logo.svg)) instead of the page-specific img.
